ConnectionsApp/src/main/java/com/myproject/connections/mapper/StateMapper.java

```diff
@@ -1,66 +1,84 @@
 package com.myproject.connections.mapper;
 
-/*@author=Shreya*/
-/*&Mapper class for States*/
 import java.lang.reflect.Type;
 import java.text.ParseException;
 import java.util.List;
-
 import org.modelmapper.ModelMapper;
 import org.modelmapper.TypeToken;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.domain.Sort;
 import org.springframework.stereotype.Component;
-
 import com.myproject.connections.entitybeans.StatesEntity;
 import com.myproject.connections.models.StatesDto;
 import com.myproject.connections.repository.StateRepository;
 
+/*@author Shreya S Jalihal
+ *Mapper class for States class
+ */
 @Component
 public class StateMapper {
-	
-    private static Logger logger = LoggerFactory.getLogger(StateMapper.class);
 
+	private static Logger logger = LoggerFactory.getLogger(StateMapper.class);
 
 	@Autowired
 	StateRepository stateRepository;
```

Comment:
why do we have a repository in a mapper? it should be in Service only . Please fix this.